EntityManager does not throw exceptions after close() in required cases